test/contlinear.jl

```diff
@@ -116,6 +116,178 @@ function contlineartest(solver::MOI.AbstractSolver, ε=Base.rtoldefault(Float64)
             @test MOI.getattribute(m, MOI.ConstraintDual(), vc2) ≈ 1 atol=ε
         end
 
+        # add new variable to get :
+        # max x + 2z
+        # s.t. x + y + z <= 1
+        # x,y,z >= 0
+
+        z = MOI.addvariable!(m)
+        push!(v, z)
+        @test v[3] == z
+
+        vc3 = MOI.addconstraint!(m, MOI.ScalarVariablewiseFunction(v[2]), MOI.GreaterThan(0.0))
+        @test MOI.getattribute(m, MOI.NumberOfConstraints{MOI.ScalarAffineFunction{Float64},MOI.GreaterThan}()) == 3
+
+        cf2 = MOI.ScalarAffineFunction(v, [1.0,1.0,1.0], 0.0)
```

Comment:
This test is meant to test column generation, so I'd say it's more appropriate to use ``ScalarCoeffficientChange`` instead of changing the entire function. This test is correct though. We should keep this version and add a second version with `ScalarCoefficientChange`.